src/data/contributors/KaranSantra.json

```diff
@@ -0,0 +1,9 @@
+{
+    "name": "Karan Santra",
+    "skills" : ["Java", "Node JS"],
+    "cause" : "creating oppurtunities for everyone",
+    "image" : "Avatar5",
+    "github" : "https://github.com/KaranSantra",
+    "twitter" : "https://twitter.com/karan_santra",
+    "linkedin" : "https://www.linkedin.com/in/karan-santra-42633511a"
+}
```

Comment:
Here too :)

Reply:
I do not understand what changes do I need to make in this file.

Reply:
ok you want me to add a blank space here as well. Got it.